Backend/Models/Item.cs

```diff
@@ -0,0 +1,10 @@
+namespace Backend1.Models;
+
+public class Item
+{
+    public uint BusinessId { get; set; }
```

Comment:
So -- I think ideally, we would have a separate table or maybe even a separate database for each business.
However, this isn't really practical.

So the approach I opted for is one global table that has a unique identifier for each business.

Whenever we add registration, that's where this business ID should come from.  Probably some sort of hash.